dotcom-rendering/scripts/check-node-versions.mjs

```diff
@@ -35,7 +35,8 @@ const requiredNodeVersionMatches =
 		},
 		{
 			filepath: 'scripts/deploy/riff-raff.yaml',
-			pattern: /^ +Recipe: dotcom-rendering.*-node-(\d+\.\d+\.\d+)$/m,
+			pattern:
+				/^ +Recipe: dotcom-rendering.*-node-(\d+\.\d+\.\d+)(-v.*)?$/m,
```

Comment:
`(-v.*)?` allows an optional `-v*` postfix

Reply:
do we want this version to be anything but a series of numbers?

```suggestion
				/^ +Recipe: dotcom-rendering.*-node-(\d+\.\d+\.\d+)(-v\d+)?$/m,
```

Reply:
otherwise I would recommend naming this `dotcom-rendering-ARM-jammy-node-18.18.2-v2-the-one-where-we-removed-pm2`

Reply:
I left it open but you're right we could have named this more descriptively. I'll merge this as is and if the need comes up again we can add a description.